tests/codeception/backend/functional/LoginCept.php

```diff
@@ -17,7 +17,6 @@
 $I->see('Password cannot be blank.', '.help-block');
 
 $I->amGoingTo('try to login with wrong credentials');
-$I->expectTo('see validations errors');
 $loginPage->login('admin', 'wrong');
 $I->expectTo('see validations errors');
```

Comment:
IMHO, we should keep the second one for to be more natural. After amGoingTo(), it must be followed by an action, in this case is login with wrong credentials then we expect something happened afterwards.